tools/readthedocs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx==5.0.2
```

Comment:
Do these need to be in sync with the Docker image versions?

Reply:
Yes, ideally - the CI pipeline builds readthedocs (`00-documentation`) in parallel with the newly enabled RTD hook on pull requests, and the two should use a consistent environment.